go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/giantswarm/microkit v1.0.0
 	github.com/giantswarm/micrologger v0.6.0
 	github.com/giantswarm/operatorkit/v6 v6.1.0
+	github.com/google/go-cmp v0.5.7
```

Comment:
Is this really needed as a new dependency?

Reply:
It is used in tests, maybe I could construct tests to avoid using it, but on the other hand it is commonly used library across our projects, so I saw no problem importing it